tests/system-tests/rdscore/internal/rdscorecommon/egress-ip.go

```diff
@@ -0,0 +1,705 @@
+}
+
+func verifyEgressIPConnectivityThreeNodes(isIPv6 bool) {
+	By("Spawning the prober pods on the EgressIP assignable hosts")
+
+	expectedIPs, err := getExpectedEgressIPList()
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to retrieve configured eIP addresses list from the egressIP %s: %v",
+			RDSCoreConfig.EgressIPName, err))
+
+	proberPodObjects, err := pod.List(APIClient, RDSCoreConfig.EgressIPNamespaceOne, egressIPPodSelector)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to retrieve prober pods list from namespace %s with label %s: %v",
+			RDSCoreConfig.EgressIPNamespaceOne, RDSCoreConfig.EgressIPPodLabel, err))
+
+	err = sendTrafficCheckIP(proberPodObjects, isIPv6, expectedIPs)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Server response was note received: %v", err))
+
+	proberPodObjects, err = pod.List(APIClient, RDSCoreConfig.EgressIPNamespaceTwo, egressIPPodSelector)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to retrieve prober pods list from namespace %s with label %s: %v",
+			RDSCoreConfig.EgressIPNamespaceTwo, RDSCoreConfig.EgressIPPodLabel, err))
+
+	err = sendTrafficCheckIP(proberPodObjects, isIPv6, expectedIPs)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Server response was not received: %v", err))
+}
+
+// VerifyEgressIPConnectivityThreeNodesIPv4 verifies egress traffic works with egressIP
+// applied for the external target.
+func VerifyEgressIPConnectivityThreeNodesIPv4() {
+	verifyEgressIPConnectivityThreeNodes(false)
+}
+
+// VerifyEgressIPConnectivityThreeNodesIPv6 verifies egress traffic works with egressIP
+// applied for the external target.
+func VerifyEgressIPConnectivityThreeNodesIPv6() {
+	verifyEgressIPConnectivityThreeNodes(true)
+}
+
+// VerifyEgressIPForPodWithWrongLabel verifies egress traffic applies only for pods with
+// the correct pod label defined.
+func VerifyEgressIPForPodWithWrongLabel() {
+	By("Spawning the prober pods on the EgressIP assignable hosts")
+
+	expectedIPs, err := getExpectedEgressIPList()
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to retrieve configured eIP addresses list from the egressIP %s: %v",
+			RDSCoreConfig.EgressIPName, err))
+
+	proberPodObjects, err := pod.List(APIClient, RDSCoreConfig.EgressIPNamespaceOne, egressIPPodSelector)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to retrieve prober pods list from namespace %s with label %s: %v",
+			RDSCoreConfig.EgressIPNamespaceOne, RDSCoreConfig.EgressIPPodLabel, err))
+
+	err = sendTrafficCheckIP(proberPodObjects, false, expectedIPs)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Server response was note received: %v", err))
+
+	proberPodObjects, err = pod.List(APIClient, RDSCoreConfig.EgressIPNamespaceOne, nonEgressIPPodSelector)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to retrieve prober pods list from namespace %s with label %s: %v",
+			RDSCoreConfig.EgressIPNamespaceTwo, RDSCoreConfig.EgressIPPodLabel, err))
```

Comment:
The `List` references `RDSCoreConfig.EgressIPNamespaceOne` and `nonEgressIPPodSelector`, shouldn't formatting be updated like:
```suggestion
			RDSCoreConfig.EgressIPNamespaceOne, nonEgressIPPodSelector, err))
```